blazar/manager/enforcement.py

```diff
@@ -153,6 +144,35 @@ def remove_lease_exception(self, user_name):
     def setup_usage_enforcement(self, project_name):
         self.initialize_project_allocation(project_name)
 
+    def check_user_max_leases(self, lease_values, resource_type):
+        user_leases = [
+            l for l in db_api.lease_get_all_by_user(lease_values['user_id'])
+            if l['status'] in [status.lease.ACTIVE, status.lease.PENDING]]
+
+        leases_by_resource_type = [
+            l['id'] for l in user_leases
+            if any([
+                x['resource_type'] == resource_type for x
+                in l['reservations']])
+        ]
+
+        existing = len(leases_by_resource_type)
+        max = CONF.enforcement.default_max_leases_by_resource_type.get(
+            resource_type, -1)
+        exempt_project = self._is_exempt_project(lease_values['project_id'])
+
+        if (existing >= max) and (max != -1) and not exempt_project:
```

Comment:
SUUUUPER nitpicky, but with the above changes, this reads like:

```python
if (num_existing >= max_leases) and (max_leases != -1) and not is_exempt_project:
```

... which reads a bit more clearly (to me.)